overlay/src/components/organisms/widgets/Scoreboard.js

```diff
@@ -92,19 +92,45 @@ const TeamTaskColor = Object.freeze({
     [TeamTaskStatus.first]: VERDICT_OK,
 });
 
-const ScoreboardTaskCell = ({ status, attempts }) => {
+const ScoreboardICPCTaskCell = ({ status, attempts }) => {
     return <ScoreboardTaskCellWrap background={TeamTaskColor[status]}>
         {status === TeamTaskStatus.first && <StarIcon/>}
         {TeamTaskSymbol[status]}
         {status !== TeamTaskStatus.untouched && attempts > 0 && attempts}
     </ScoreboardTaskCellWrap>;
 };
 
-ScoreboardTaskCell.propTypes = {
+const ScoreboardScoreTaskCell = ({ status, score, attempts }) => {
+    return <ScoreboardTaskCellWrap background={TeamTaskColor[status]}>
+        {status === TeamTaskStatus.first && <StarIcon/>}
+        {TeamTaskStatus === TeamTaskStatus.unknown ? "?" : score}
+        {status !== TeamTaskStatus.untouched && attempts > 0 && attempts}
```

Comment:
If a team makes n submissions and gets m score it will be rendered as "mn" without a separator resulting in confusion as to why a score could be more than 100

Reply:
<img width="386" alt="image" src="https://user-images.githubusercontent.com/8657078/201534896-af8a3131-762c-44d9-bd7d-613323a4a02b.png">
from https://www.youtube.com/watch?v=or5kBuwBxJo&list=PLn4CTt5ibU6c-ykPIbX5OW6TuqxmDL3NK&index=1

Reply:
Fixed in https://github.com/icpc/live-v3/pull/81/commits/ac0e68634fcc9c189456e5f350e638d0b2abacdd